src/test/java/org/opendevstack/provision/services/JenkinsPipelineAdapterTest.java

```diff
@@ -89,10 +91,13 @@ private JenkinsPipelineProperties buildJenkinsPipelineProperties() {
     jenkinsPipelineProperties.addQuickstarter(
         adminjobQuickstarter(
             "create-projects",
-            "opendevstack/ods-core.git#production/create-projects/Jenkinsfile",
-            "internal quickstarter for creating new initiatives"));
+            "ods-core.git",
+            "internal quickstarter for creating new initiatives",
+            Optional.of("production"),
+            Optional.of("/create-projects/Jenkinsfile")));
```

Comment:
This would usually be without leading slash - so just `create-projects/Jenkinsfile`, right?